### ISSUE TITLE
Cursor disapears in address bar

### ISSUE THREAD

**Issue by pierreporte** [OtterBrowser/otter-browser#1008]
The blinking cursor in the address bar disappears when the URL completion menu appears. This keeps me from editing the entered text on the position where I click. It is still possible to move the cursor (which is then invisible) with the arrow keys.

For example, I enter “otterbrowser.org”.Wanting to add the missing dash, I click between the first “r” and the “b” and enter “-” and I get “otterbrowser.org-”

*Version:  d69aed0 or cc6b5b5 on up to date Archlinux.*

**Comment by Emdek** [comment]
@pierreporte, yes, this is known limitation of current solution, hopefully there is a way to fix it properly.

**Comment by Zero3K** [comment]
This issue is still present in the latest weekly snapshot of it.

**Comment by Zero3K** [comment]
Its still occurring in the current weekly.
